test/setupTests.ts

```diff
@@ -1,6 +1,21 @@
+import 'styled-components'
+
 import '@testing-library/jest-dom/extend-expect'
 import 'jest-styled-components'
 
+window.matchMedia = jest.fn().mockImplementation(query => {
+  return {
+    addEventListener: jest.fn(),
+    addListener: jest.fn(), // deprecated
+    dispatchEvent: jest.fn(),
+    matches: false,
+    media: query,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)